### PR TITLE
New version: Arc.Arc version 0.12.0.13667

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/144049)

### CODE REVIEW

**Comment by SpecterShell** [comment]
Shouldn't the identifier be `TheBrowserCompany.Arc`?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-03-13T00:21:11.9897028Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][1] Description: Missing required property 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #1 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: Arc.Arc.installer.yaml Manifest Error: Required field missing. [InstallerSha256]

(Automated response - build 870.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by Exorcism0666** [comment]
> The identifier could be renamed to `TheBrowserCompany.Arc` according to the msix manifest. @Exorcism0666

I will do this when this version is published

File: manifests/a/Arc/Arc/0.12.0.13667/Arc.Arc.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.4
+PackageFamilyName: TheBrowserCompany.Arc_ttt1ap7aakyb4
+ReleaseDate: 2024-03-07
+Installers:
+- Architecture: x64
+  InstallerUrl: https://releases.arc.net/windows/prod/0.12.0.13667/Arc.x64.msix
+  InstallerSha256: A0CA1DC369B4BA8909A1964A064B718EAE213E5808EA7E075F3F7F52362C8C09
+  SignatureSha256: 37692D5182F9CEDE1B5AAB404CFE449140474A5FA1AE6ED3BDD0B60F35D732E7
+- Architecture: arm64
+  InstallerUrl: https://releases.arc.net/windows/prod/0.12.0.13667/Arc.arm64.msix
```
Comment (stephengillie) [inline manifests/a/Arc/Arc/0.12.0.13667/Arc.Arc.installer.yaml:23]:
```suggestion
  InstallerUrl: https://releases.arc.net/windows/prod/0.12.0.13667/Arc.arm64.msix
  InstallerSha256: 13E994AD9EAAE468F02350270C39C1FBCD73E33A98DB0FD9BF4E9C44891AEB52
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Signature Update Failed.
> Failure to parse win 10, attempting win 11
> 2024-03-14 06:32:16.857 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(722)\WindowsPackageManager.dll!00007FF8BF0FF3CF: (caller: 00007FF8BEE884FF) LogHr(1) tid(1ce4) 8A15005E     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FF8BF004DD8: (caller: 00007FF8BF0AF820) Exception(1) tid(1ce4) 8A15005E ] 
> 2024-03-14 06:32:16.857 [REPO] Failed to open available source: msstore
> 2024-03-14 06:32:19.149 [CLI ] Caught wil::ResultException: C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FF8BF004DD8: (caller: 00007FF8BF0AF820) Exception(1) tid(1ce4) 8A15005E 
> 2024-03-14 06:32:20.854 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(722)\WindowsPackageManager.dll!00007FF8BF0FF3CF: (caller: 00007FF8BEE884FF) LogHr(1) tid(dd8) 8A15005E     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FF8BF004DD8: (caller: 00007FF8BF0AF820) Exception(1) tid(dd8) 8A15005E ] 
> 2024-03-14 06:32:20.854 [REPO] Failed to open available source: msstore
> 2024-03-14 06:32:21.821 [CLI ] Caught wil::ResultException: C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FF8BF004DD8: (caller: 00007FF8BF0AF820) Exception(1) tid(dd8) 8A15005E 
> 2024-03-14 06:32:30.395 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(722)\WindowsPackageManager.dll!00007FF8C35FF3CF: (caller: 00007FF8C33884FF) LogHr(1) tid(23e0) 8A15005E     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FF8C3504DD8: (caller: 00007FF8C35AF820) Exception(1) tid(23e0) 8A15005E ] 
> 2024-03-14 06:32:30.395 [REPO] Failed to open available source: msstore
> 2024-03-14 06:34:02.716 [CORE] Deployment operation #2: Windows cannot install package TheBrowserCompany.Arc_0.12.0.13667_x64__ttt1ap7aakyb4 because this package depends on a framework that could not be found. Provide the framework "Microsoft.WindowsAppRuntime.1.4" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 4000.1082.2259.0, along with this package to install. The frameworks with name "Microsoft.WindowsAppRuntime.1.4" currently installed are: {Microsoft.WindowsAppRuntime.1.4_4000.1010.1349.0_x64__8wekyb3d8bbwe Microsoft.WindowsAppRuntime.1.4_4000.1049.117.0_x64__8wekyb3d8bbwe Microsoft.WindowsAppRuntime.1.4_4000.1049.117.0_x86__8wekyb3d8bbwe}
> 2024-03-14 06:34:02.716 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FF8C3481A8F: (caller: 00007FF8C33D947D) Exception(2) tid(23e0) 80073CF3 Package failed updates, dependency or conflict validation.
> 2024-03-14 06:34:02.805 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FF8C3480840: (caller: 00007FF8C3481E4C) Exception(3) tid(23e0) 80073CF1 Package was not found.
> 2024-03-14 06:34:02.805 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(126)\WindowsPackageManager.dll!00007FF8C35FBBE1: (caller: 00007FF8C3483363) LogHr(2) tid(23e0) 80073CF1 Package was not found.
>     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FF8C3480840: (caller: 00007FF8C3481E4C) Exception(3) tid(23e0) 80073CF1 Package was not found.
> 2024-03-14 06:34:02.847 [CLI ] MSIX installer failed: 2147958003

(Automated response - build 870.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Signature Update Failed.
> Failure to parse win 10, attempting win 11
> Installation failed with exit code -1978334960
> 2024-03-16 10:30:43.418 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(722)\WindowsPackageManager.dll!00007FFD7D2EF3CF: (caller: 00007FFD7D0784FF) LogHr(1) tid(1b34) 8A15005E     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFD7D1F4DD8: (caller: 00007FFD7D29F820) Exception(1) tid(1b34) 8A15005E ] 
> 2024-03-16 10:30:43.418 [REPO] Failed to open available source: msstore
> 2024-03-16 10:30:45.742 [CLI ] Caught wil::ResultException: C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFD7D1F4DD8: (caller: 00007FFD7D29F820) Exception(1) tid(1b34) 8A15005E 
> 2024-03-16 10:30:47.211 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(722)\WindowsPackageManager.dll!00007FFD7D2EF3CF: (caller: 00007FFD7D0784FF) LogHr(1) tid(1e90) 8A15005E     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFD7D1F4DD8: (caller: 00007FFD7D29F820) Exception(1) tid(1e90) 8A15005E ] 
> 2024-03-16 10:30:47.211 [REPO] Failed to open available source: msstore
> 2024-03-16 10:30:47.951 [CLI ] Caught wil::ResultException: C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFD7D1F4DD8: (caller: 00007FFD7D29F820) Exception(1) tid(1e90) 8A15005E 
> 2024-03-16 10:30:53.851 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(722)\WindowsPackageManager.dll!00007FFD7A6BF3CF: (caller: 00007FFD7A4484FF) LogHr(1) tid(9b4) 8A15005E     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFD7A5C4DD8: (caller: 00007FFD7A66F820) Exception(1) tid(9b4) 8A15005E ] 
> 2024-03-16 10:30:53.851 [REPO] Failed to open available source: msstore
> 2024-03-16 10:31:41.460 [CLI ] ShellExecute installer failed: 2147944126

(Automated response - build 870.)

**Pull request closed without merging**